apps/notepad/src/app/component/note-details/note-details.component.html

```diff
@@ -0,0 +1,56 @@
+<form [formGroup]="createNoteDetailForm">
+  <div>
+    <div class="flex">
+      <div class="max-w-sm rounded shadow-lg">
+        <input
+          class="text-center font-extrabold text-2xl"
+          type="text"
+          name="inputTitle"
+          formControlName="detailTitle"
+          [disabled]="!isEditable"
+        />
+        <input
+          type="text"
+          name="inputContent"
+          formControlName="detailContent"
+          [disabled]="!isEditable"
+        />
+
+        <div class="text-right font-medium">
+          {{ currentNoteDetail.createdAt | date : 'mediumDate' }}
+        </div>
+      </div>
+    </div>
+    <div>
+      <button type="button" class="btn blue" (click)="onEdit()">Edit</button>
+      <button class="btn blue" type="submit" (click)="onSubmit()">
+        submit
+      </button>
+    </div>
+
+    <div *ngFor="let todo of currentNoteDetail.todoList">
+      {{ todo.name }} {{ todo.completed }}
+    </div>
+
+    <div class="flex">
+      <div class="form-check">
+        <input
+          class="form-check-input appearance-none h-4 w-4 border border-gray-300 rounded-sm bg-white checked:bg-blue-600 checked:border-blue-600 focus:outline-none transition duration-200 mt-1 align-top bg-no-repeat bg-center bg-contain float-left mr-2 cursor-pointer"
+          type="checkbox"
+          id="flexCheckIndeterminate"
+          formControlName="detailCheckBox"
+        />
```

Comment:
detailTodo and detailCheckBox- both input fields are related and hence ideally should be grouped together in a formGroup.